spec/models/company_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+describe 'Company model' do
+  before do
+    @company = Company.new(name: 'Example Company', email: 'example@email.com')
+    @company.save
```

Comment:
use factory for this
you need to test the relations also
structure should be
describe relations ....
  describe 
  describe
describe validation ...
  describe
  describe